docs/sedonaspark.md

```diff
@@ -0,0 +1,175 @@
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+# SedonaSpark
+
+SedonaSpark extends Apache Spark with a rich set of out-of-the-box distributed Spatial Datasets and functions that efficiently load, process, and analyze large-scale spatial data across machines. SedonaSpark is an excellent option for datasets too large for a single machine.
+
+=== "SQL"
+
+    ```sql
+    SELECT superhero.name
+    FROM city, superhero
+    WHERE ST_Contains(city.geom, superhero.geom)
+    AND city.name = 'Gotham'
+    ```
+
+=== "PySpark"
+
+    ```python
+    sedona.sql(
+        """
+        SELECT superhero.name
+        FROM city, superhero
+        WHERE ST_Contains(city.geom, superhero.geom)
+        AND city.name = 'Gotham'
+    """
+    )
+    ```
+
+=== "Java"
+
+    ```java
+    Dataset<Row> result = spark.sql(
+    "SELECT superhero.name " +
+    "FROM city, superhero " +
+    "WHERE ST_Contains(city.geom, superhero.geom) " +
+    "AND city.name = 'Gotham'"
+    );
+    ```
+
+=== "Scala"
+
+    ```scala
+    sedona.sql("""
+        SELECT superhero.name
+        FROM city, superhero
+        WHERE ST_Contains(city.geom, superhero.geom)
+        AND city.name = 'Gotham'
+    """)
+    ```
+
+=== "R"
+
+    ```r
+    result <- sql("
+    SELECT superhero.name
+    FROM city, superhero
+    WHERE ST_Contains(city.geom, superhero.geom)
+    AND city.name = 'Gotham'
+    ")
+    ```
+
+## Key features
+
+* **Blazing fast**: SedonaSpark executes computations in parallel on many nodes in a cluster so that large computations can run fast.
+* Supports **various file formats**, including GeoJSON, Shapefile, GeoParquet, STAC, JDBC, OSM PBF, CSV, and PostGIS.
+* Exposes several **language APIs,** including SQL, Python, Java, Scala, and R.
+* **Scalable**: Horizontally scale to tens, hundreds, or thousands of nodes depending on the size of your data.  You can process massive spatial datasets with SedonaSpark.
+* **Portable**: Easy to run in a custom environment, locally or in the cloud with AWS EMR, Microsoft Fabric, or Google DataProc.
+* **Extensible**: You can extend SedonaSpark with your custom logic that suits your specific geospatial data analysis needs.
+* **Open source**: Apache Sedona is an open-source project managed in accordance with the Apache Software Foundation's guidelines.
+* Extra functionality like [nearest neighbor searching](https://sedona.apache.org/latest/api/sql/NearestNeighbourSearching/) and geostats like [DBSCAN](https://sedona.apache.org/latest/tutorial/sql/#cluster-with-dbscan)
+
+## Installation
+
+Here’s how to install SedonaSpark with various build tools:
+
+=== "pip"
+
+    ```bash
+    pip install apache-sedona
+    ```
+
+=== "SBT"
+
+    ```
+    libraryDependencies += "org.apache.sedona" % "sedona-common" % "{{ sedona.current_version }}"
```

Comment:
the installation of SedonaSpark is much more complex than this. Should we instead direct users to the installation page instead?

Reply:
Yea, I just removed these summary install instructions.  Users can easily navigate to the "Install" section, which includes full details for each runtime.